OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/SSAIPlayerViewController.m

```diff
@@ -0,0 +1,144 @@
+//
+//  SSAIPlayerViewController.m
+//  OoyalaSkinSampleApp
+//
+//  Copyright © 2018 Ooyala, Inc. All rights reserved.
+//
+
+#import "SSAIPlayerViewController.h"
+#import <OoyalaSDK/OoyalaSDK.h>
+#import <OoyalaSSAISDK/OoyalaSSAISDK.h>
+#import <OoyalaSkinSDK/OoyalaSkinSDK.h>
+#import "AppDelegate.h"
+
+@interface SSAIPlayerViewController ()
+
+#pragma mark - Private properties
+
+@property (nonatomic, retain) OOSsaiPlugin *ssaiPlugin;
+@property (nonatomic, retain) OOSkinViewController *skinController;
+@property (nonatomic, retain) OOOoyalaPlayer *ooyalaPlayer;
+@property NSString *embedCode;
+@property NSString *nib;
+@property NSString *pcode;
+@property NSString *playerDomain;
+
+@end
+
+@implementation SSAIPlayerViewController
+
+AppDelegate *appDel;
+
+#pragma mark - Initialization
+
+- (id)initWithPlayerSelectionOption:(PlayerSelectionOption *)playerSelectionOption qaModeEnabled:(BOOL)qaModeEnabled {
+  self = [super initWithPlayerSelectionOption: playerSelectionOption qaModeEnabled:qaModeEnabled];
+  if (self.playerSelectionOption) {
+    self.nib = self.playerSelectionOption.nib;
+    self.embedCode = self.playerSelectionOption.embedCode;
+    self.title = self.playerSelectionOption.title;
+    self.playerDomain = self.playerSelectionOption.playerDomain;
+    self.pcode = playerSelectionOption.pcode;
+  }
+  return self;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  
+  appDel = (AppDelegate *)[[UIApplication sharedApplication] delegate];
+  
+  
+  // Create Ooyala ViewController
+  OOOptions *options = [OOOptions new];
+  self.ooyalaPlayer = [[OOOoyalaPlayer alloc] initWithPcode:self.pcode
+                                                     domain:[[OOPlayerDomain alloc] initWithString:self.playerDomain]
+                                                    options:options];
+  
+  NSURL *jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+  //  NSURL *jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios"];
```

Comment:
Can you put a comment that you should uncomment this for local debugging?

Reply:
done

Reply:
done